include/chef_types.hrl

```diff
@@ -219,15 +220,20 @@
 %% Not a true chef object either, keys are owned by their associated user
 %% or client object and are managed in that context.
 -record(chef_key, {
-	  'id'  :: object_id(),     %% guid of user or client, unique with key_name
-	  'key_name',               %% user named string describing key
-	  'public_key',             %% PKCS#1 public key or a certificate
-	  'key_version',            %% 0 for public_key, 1 for cert
-	  'expires_at' :: binary(), %% expiration time (utc)
-	  'last_updated_by' :: binary(), %% authz id of updater
-	  'created_at' :: binary(), %% when created
-	  'updated_at' :: binary()  %% when last updated
-	 }).
+          'id'  :: object_id(),     %% guid of user or client, unique with key_name
+          'key_name',               %% user named string describing key
+          'public_key',             %% PKCS#1 public key or a certificate
+          'key_version',            %% 0 for public_key, 1 for cert
+          'expires_at' :: binary(), %% expiration time (utc)
+          'last_updated_by' :: binary(), %% authz id of updater
+          'created_at' :: binary(), %% when created
+          'updated_at' :: binary(),  %% when last updated
+
+          %% Important note - in 'chef_key:flatten' we explcitly drop this last field
```

Comment:
Renaming "flatten" to "fields_for_insert" will make the intent clear, and discouraging use of flatten - I think explicit is better anyway, since it maps to the parameters of a specifically defined query.  As far as I've seen, that's the only thing that flatten gets used for.